docker-compose.yml

```diff
@@ -0,0 +1,116 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+version: '3.5'
+
+services:
+    trafficops:
+        build:
+            context: dev/traffic_ops/
+        depends_on:
+            - db
+        domainname: dev.ciab.test
+        hostname: trafficops
+        image: trafficops-dev
+        ports:
+            - 6443:443
+            - 6444:6444
+        volumes:
+            - .:/root/go/src/github.com/apache/trafficcontrol/
+
+    db:
+        image: postgres:13.2-alpine
+        ports:
+            - 5432:5432
+        environment:
+            - POSTGRES_PASSWORD=twelve12
+        hostname: db
+        domainname: dev.ciab.test
+
+    trafficportal:
+        build:
+            context: dev/traffic_portal
+        depends_on:
+            - trafficops
+        hostname: trafficportal
+        domainname: dev.ciab.test
+        image: trafficportal-dev
+        ports:
+            - 444:443
+        volumes:
+            - .:/root/go/src/github.com/apache/trafficcontrol
+
+    tpv2:
+        build:
+            context: dev/tpv2
+        depends_on:
+            - trafficops
+        hostname: tpv2
+        domainname: dev.ciab.test
+        image: tpv2-dev
+        ports:
+            - 443:443
+        volumes:
+            - .:/root/go/src/github.com/apache/trafficcontrol
+
+    trafficmonitor:
+        build:
+            context: dev/traffic_monitor
+        depends_on:
+            - trafficops
+        hostname: trafficmonitor
+        domainname: dev.ciab.test
```

Comment:
Instead of specifying the domain, add it the `trafficmonitor` service to a network:
```yml
        networks:
          - ciab
```